diff --git a/bsps/powerpc/beatnik/include/bsp/VMEConfig.h b/bsps/powerpc/beatnik/include/bsp/VMEConfig.h
index 986a01674e..535abc1998 100644
--- a/bsps/powerpc/beatnik/include/bsp/VMEConfig.h
+++ b/bsps/powerpc/beatnik/include/bsp/VMEConfig.h
@@ -67,7 +67,8 @@
 /* start of the A32 window on the VME bus
  * TODO: this should perhaps be a configuration option
  */
-#define _VME_A32_WIN0_ON_VME  0x20000000
+//#define _VME_A32_WIN0_ON_VME  0x20000000
+#define _VME_A32_WIN0_ON_VME  0x00800000

 /* if _VME_DRAM_OFFSET is defined, the BSP
  * will map our RAM onto the VME bus, starting
